HACKING.md

```diff
@@ -116,18 +117,18 @@ And setting these options will not change the output.
 See “[Query Screen](https://solr.apache.org/guide/7_7/query-screen.html#query-screen)”
 in Apache Solr Reference Guide 7.7 for more information.
 
-#### Parameters priority
+#### Parameter priority
```

Comment:
I wasn’t sure here because priority makes sense when there are at least two parameters (or two settings for the same parameter).

Reply:
I *think* in English they often use it in singular when written this way? Like "operator priority", but "priority of operators". But Alastair can correct me if I'm wrong :)